docs/features/kill-persistence.md

```
@@ -218,6 +223,35 @@ end
+
+1. A Docker Compose configuration (`docker-compose.yml`) defines both the app container and an optional Postgres container with a profile.
+2. The devcontainer.json file is configured to use this Docker Compose setup.
+3. The Postgres container is only started when the `persistence` profile is activated.
+4. Persistence can be enabled by uncommenting the `ENABLE_PERSISTENCE` environment variable in the devcontainer.json file.
+
+### Using Persistence in Development
+
+To enable persistence during development:
+
+1. Edit `.devcontainer/devcontainer.json` and uncomment:
+   ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)